### DISCUSSION TITLE
403 error when pushing to Repos in new Organisations

### DISCUSSION THREAD

**Discussion by IanMayo** [community/community#22125] [New to GitHub]
<p>I have two organisations, and can create repos within them, and push to these repos.</p>
<p>For any new organisations I create, I can make edits via <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>, but when I try to push local changes I get:</p>
<p>remote: Permission to [new-org]/[new-repo].git denied to [user-name].<br>
fatal: unable to access ‘<a href="https://github.com/%5Bnew-org%5D/%5Bnew-repo%5D.git/" target="_blank" rel="nofollow noopener">https://github.com/[new-org]/[new-repo].git/</a>’: The requested URL returned error: 403</p>
<p>I get the above issue if I try to push from command line or from SmartGit.</p>
<p>Pushing to repos from my existing organisations works fine in both cases.</p>

**Comment by mpboom** [comment]
<p>Did you try doing this on different networks? Are you using the correct account locally?</p>

**Comment by IanMayo** [comment]
<p>Aah, good suggestion.  I different network may be a challenge, but I can certainly try it from a different PC.</p>

**Comment by IanMayo** [comment]
<p>Ok, from a different PC I get the same 403 denied error when I push to a repo under the new organisations.  But, I can push changes from the older organisations.</p>

**Comment by IanMayo** [comment]
<p>Ok. I’ve just run a test from a MS Windows VM, and get the same problem.  Can push to repos from old orgs, but not the new ones.</p>

**Comment by IanMayo** [comment, marked as answer]
<p>Ok, I was running a further trial, on a Linux VM.</p>
<p>I installed GitKraken, and in the GitHub authorisation process I had to indicate when organisations it could access.</p>
<p>The new repo had a red Cross against it. I selected it, to allow GitKraken access to that organisation. GitKraken was then able to push to the new repo.</p>
<p>I then received an email telling me the GitHub authorisation had been added. It included a link to this page <a href="https://github.com/settings/connections/applications/" rel="nofollow noopener">https://github.com/settings/connections/applications/</a></p>
<p>From that page I was able to view authorised OAuth apps. SmartGit (my normal Git client was listed). When I went into the page for that app, I saw it wasn’t allowed access to my new organisation. I authorised access.</p>
<p>From that point on, I was able to push from either SmartGit or the command-line.</p>
<p>Fixed!</p>

**Reply by bluesman80** [reply]
I had a similar problem with IntelliJ IDEA. I could not push to the repository from within IntelliJ IDEA. After granting access to it on the webpage you shared, it worked. Apparently, Github requires explicitly granting access to tools.

**Comment by mpboom** [comment]
<p>Wow, that is some nice investigation! Glad you got it fixed!</p>

**Comment by Adigo10** [comment]
<p>Step 1: Add your PC/Mac’s SSH key to your Github Account<br>
Refer - <a href="https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account" rel="nofollow noopener">https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account</a><br>
Step2: Clone with SSH instead of HTTPS</p>

**Comment by Zahnentferner** [comment]
Thank you so much!
